### DISCUSSION TITLE
From telegram import Bot (Import Error)

### DISCUSSION THREAD

**Discussion by theresarto** [python-telegram-bot/python-telegram-bot#4404] [Q&A]
I have uninstalled and reinstalled the Telegram package through pip install multiple times.

However, it cannot find any class such as Bot, Update, etc. Everytime I run my code, this error comes up:

AttributeError: module 'telegram' has no attribute 'Bot'

and

    from telegram import Bot
ImportError: cannot import name 'Bot' from 'telegram' (unknown location)

Can someone help me?

**Comment by Poolitzer** [comment]
Hi, we have a wiki FAQ entry for that: https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#why-am-i-getting-importerror-cannot-import-name-xy-from-telegram

**Comment by MaugerMK** [comment]
from telegram import Bot

**Comment by MaugerMK** [comment]
![1722480393785](https://github.com/user-attachments/assets/3d948de0-49b7-4cf5-befb-5ed0b0b2182b)

**Comment by MaugerMK** [comment]
![1722480366805](https://github.com/user-attachments/assets/e2cd9fde-9285-47f0-bdfc-7b12a0ba8e12)

**Comment by MaugerMK** [comment]
![1722480340113](https://github.com/user-attachments/assets/d66d5e7c-59cc-4852-bebb-c71409e4b3b5)

**Comment by MaugerMK** [comment]
Je veux créer un bot

**Comment by MaugerMK** [comment]
![Uploading 1722480387248.jpg…]()

**Comment by MaugerMK** [comment]
![Uploading 1722480387248.jpg…]()

**Comment by MaugerMK** [comment]
![1722480358205](https://github.com/user-attachments/assets/0cf82ceb-484f-46a0-a680-49800c334908)

**Comment by MaugerMK** [comment]
![1722480307089](https://github.com/user-attachments/assets/c1375feb-0696-4327-b8a8-8b8e3c6db369)

**Comment by MaugerMK** [comment]
![Uploading 1722480294149.jpg…]()

**Comment by MaugerMK** [comment]
![Uploading 1722480281760.jpg…]()
